Resources/Prototypes/Nyanotrasen/psionicPowers.yml

```
@@ -8,6 +8,9 @@
     XenoglossyPower: 0.75
     PsychognomyPower: 0.75
     MassSleepPower: 0.3
+    NoosphericZapPower: 0.5
+    PyrokinesisPower: 0.5
+    EtherealVisionPower: 0.5
```

```suggestion
    NoosphericZapPower: 0.15
    PyrokinesisPower: 0.15
```

Due that those powers are dangerous move them to 0.15.

EtherealVisionPower is a fallback usedby Shadowkin, this is not a psionic power.

---

Note: PyrokinesisPower is missing some psionic actions.
add the follow code between line 93 - 94 in [Resources/Prototypes/Actions/psionics.yml](https://github.com/Fansana/floofstation1/pull/487/files#diff-791fdaa97be7cd713802999aec5f8786d523abcda387c6b826d748d3d440105d)
```
 blacklist:
        components:
          - PsionicInsulation
          - Mindbroken
```